core/src/main/java/org/mule/runtime/core/internal/profiling/tracing/event/span/InternalSpan.java

```diff
@@ -33,6 +35,12 @@ public interface InternalSpan extends Span {
    */
   void end();
 
+
+  /**
+   * Ends the span, previously recording the provided error.
```

Comment:
Why we should end the span when an error is registered? I would think we should

Reply:
I think this is only a jdoc comment.

Reply:
It is not ending the Span, it was just the JDoc.